core/modules/layout/layout.theme.inc

```diff
@@ -456,6 +456,10 @@ function template_preprocess_block(&$variables) {
     }
     else {
       $variables['classes'][] = 'block-hero-no-image';
+      // @todo do a better version check here.
+      if (version_compare(BACKDROP_VERSION, '1.29.99', '>=')) {
```

Comment:
@stpaultim this does a (now sloppy) version check on the Backdrop version and adds a CSS class only if it's a newer version. This does not yet check, whether this is a new install or installed before.
That requires some `state_get('install_time')` check or so.

I don't have the time to dig into that right now, so please understand, that this is just a POC, no final, elaborated code, yet.